tests/test_boot.py

```diff
@@ -40,6 +40,31 @@ def test_boot(config: Image.Config, format: OutputFormat) -> None:
             args = ["systemd.mask=systemd-resolved.service"] if format == OutputFormat.directory else []
             image.boot(options=options, args=args)
 
+        if format == OutputFormat.oci:
+            if image.config.distribution in (Distribution.arch, Distribution.centos, Distribution.debian):
+                pytest.skip("These distro's systemd version does not properly run inside the podman")
+
+            # These services fail as /etc/machine-id cannot be properly initialized.
+            # This is not possible as overmounting is not permitted inside the container.
+            # Running the container with higher privileges using `--privileged` or `--cap-add SYS_ADMIN`
```

Comment:
Please don't use backticks. Backticks only make sense in markdown, and this is not markdown.